src/PowerShell/Help/Microsoft.WinGet.Client/Export-WinGetPackage.md

```diff
@@ -0,0 +1,383 @@
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -MatchOption
+
+Specify the match option for a WinGet package query
+
+```yaml
+Type: Microsoft.WinGet.Client.PSObjects.PSPackageFieldMatchOption
+Parameter Sets: FoundSet
+Aliases:
+Accepted values: Equals, EqualsCaseInsensitive, StartsWithCaseInsensitive, ContainsCaseInsensitive
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -Moniker
+
+Specify the WinGet package moniker
+
+```yaml
+Type: System.String
+Parameter Sets: FoundSet
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -Name
+
+Specify the WinGet package name
+
+```yaml
+Type: System.String
+Parameter Sets: FoundSet
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -PSCatalogPackage
+
+Specify PSCatalogPackage
+
+```yaml
+Type: Microsoft.WinGet.Client.Engine.PSObjects.PSCatalogPackage
+Parameter Sets: GivenSet
+Aliases: InputObject
+
+Required: False
+Position: 0
+Default value: None
+Accept pipeline input: True (ByPropertyName, ByValue)
+Accept wildcard characters: False
+```
+
+### -Query
+
+Search string for WinGet package. The query parameter will be used to search across multiple
+properties in a WinGet package manifest. Fields include the package name, package identifier,
+moniker, and tags.
+
+```yaml
+Type: System.String[]
+Parameter Sets: FoundSet
+Aliases:
+
+Required: False
+Position: 0
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -Scope
+
+Specify WinGet package installer scope
+
+> [!NOTE]
+> The installer scope must be available in the WinGet package manifest.
+
+```yaml
+Type: Microsoft.WinGet.Client.PSObjects.PSPackageInstallScope
+Parameter Sets: (All)
+Aliases:
+Accepted values: Any, User, System, UserOrUnknown, SystemOrUnknown
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -SkipDependencies
+
+Specifies the WinGet package dependencies should not be downloaded.
+
+```yaml
+Type: System.Management.Automation.SwitchParameter
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -Source
+
+Specifies WinGet source.
+
+```yaml
+Type: System.String
+Parameter Sets: FoundSet
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -Version
+
+Specifies the WinGet package version to download
```

Comment:
What is the format of the version string, semvar? Can you specify a prerelease version?
```suggestion
Specifies the version of the WinGet package to download.
```